src/app/ui/components/dowpdown-menu/dropdown-menu.stories.tsx

```diff
@@ -1,38 +1,55 @@
-import * as RadixDropdownMenu from '@radix-ui/react-dropdown-menu';
 import type { Meta, StoryObj } from '@storybook/react';
+import { HStack, styled } from 'leather-styles/jsx';
 
-import { ListItemType } from '../list/list-item';
-import { DropdownMenu as Component } from './dropdown-menu';
-import { DropdownMenuItem } from './dropdown-menu-item';
-import { DropdownMenuSectionLabel } from './dropdown-menu-section-label';
+import { ChevronDownIcon } from '../icons/chevron-down-icon';
+import { PlaceholderIcon } from '../icons/placeholder-icon';
+import { DropdownMenu as Component, DropdownMenuItem } from './dropdown-menu';
+import { DropdownMenuItemLayout } from './dropdown-menu-item.layout';
 
-const items: ListItemType[] = [{ label: 'Label 1' }, { label: 'Label 2' }];
+const items: DropdownMenuItem[] = [{ label: 'Label 1' }, { label: 'Label 2' }];
 
-const meta: Meta<typeof Component> = {
-  component: Component,
+const meta: Meta<typeof Component.Root> = {
+  component: Component.Root,
   tags: ['autodocs'],
   title: 'Dropdown Menu',
 };
 
 export default meta;
-type Story = StoryObj<typeof Component>;
+type Story = StoryObj<typeof Component.Root>;
 
 export const DropdownMenu: Story = {
   render: () => (
-    <Component>
-      <RadixDropdownMenu.Group>
-        <DropdownMenuSectionLabel label="Items" />
-        {items.map(item => {
-          return (
-            <DropdownMenuItem
-              key={item.label}
-              iconLeft={item.iconLeft}
-              iconRight={item.iconRight}
-              label={item.label}
-            />
-          );
-        })}
-      </RadixDropdownMenu.Group>
-    </Component>
+    <Component.Root>
+      <Component.Trigger>
+        <styled.button>
+          <HStack gap="space.02" width="100%">
+            <styled.span textStyle="label.02">Options</styled.span>
+            <ChevronDownIcon />
+          </HStack>
+        </styled.button>
+      </Component.Trigger>
+      <Component.Portal>
+        <Component.Content align="start" sideOffset={8}>
+          <Component.Group>
+            <Component.Label>Label</Component.Label>
+            {items.map(item => {
+              return (
```

Comment:
You can implicitly return here if you want to get rid of this `return`. Not a big deal at all though